tests/app/forms/validation/test_date_check_validator.py

```diff
@@ -24,9 +24,11 @@ def test_date_type_validator_empty_string(self):
     def test_date_type_validator_missing_day(self):
         validator = DateCheck()
 
-        mock_form = Mock()
+        mock_form = MagicMock()
```

Comment:
Why have these changed from `Mock` to `MagicMock`?

Reply:
From what I saw in debug we cannot iterate through elements of `Mock` which breaks our code. In `MagicMock` we don't have to declare extra methods we want to use.

Reply:
Any ones in particular? I tried changing some instances of this back to `Mock` and they worked.

Reply:
`MagicMock` is removed now, after applying Mebin's fix it's not needed anymore. Thanks👍